.github/workflows/prow_commands.yml

```diff
@@ -1,43 +0,0 @@
-# Copyright 2022 The Serverless Workflow Specification Authors
```

Comment:
why removing these?

Reply:
not being used, the action is outdated, and GH keeps complaining about permissions. If you know which permissions to give to this pipelines, I can keep them.

Reply:
I didn't notice any issue related to permissions, I've compared it with another repository that uses this very same prow and the token permissions are the same.

But I have no strong opinion about removing and keeping it :)

/approve

Reply:
These permissions here:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token

Do you know which?